README.md

```diff
@@ -53,18 +53,62 @@ const HelloWorld = () => <h1>Hello, Birst</h1>;
 
 embedInBirst(() => <HelloWorld />);
 ```
+## Local Development and Debugging
+Working on a project within a Birst dashboard itself is a pretty horrendous experience.  I recommend using `webpack-dev-server` with hot reloading for the majority of development and then give it a final test in Birst before shipping!  You can even combine `webpack-dev-server` with `ngrok` to host your local dev content in a Birst dashboard directly.
+
+To test and develop locally just start up `webpack-dev-server` (`Ctrl+Shift+B` if you're using VSCode). This will serve the content to the specified port on local host. By default this is `http://localhost:3000/index.html`.
+
+Navigate to the `index.html` page for the selected component (example: `localhost:3000/src/my-component/my-component.html`). Webpack will automatically rebuild any changes made to the TSX or HTML for that component.
+
+> Note: If you are running locally _without_ ngrok (i.e. not embedded within Birst) then you don't have access to the Birst API. To improve the dev experience, whenever `process.env.NODE_ENV !== 'production'` this library will replace the _real_ BQL API with a mock one that will use `fetch` to resolve the `sampleResponseUrl` for your defined query. It will error with `Unexpected Token at position x` if you try and host locally without a valid `sample.json`.
```

Comment:
What do you think about moving this last sentence under a "common problems" section?  It might fit better there, along with the (presumably) other 50 "common" issues we find!